programmers/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,33 @@
+function solution(s){
+    var count_p = 0;
+//s내부의 p의 개수를 세줄 변수를 선언하였습니다.    
+    var search_p = 'p';
+// indexof라는 함수를 이용하여 p를 찾기 위하여 search_p라는 문자 p를 담고있는 변수를 선언하였습니다. 
+    var index_p = s.indexOf(search_p);
+// indexof라는 함수를 사용하여 s내부의 p의 위치를 입력받는 index_p를 선언하였습니다.
+    while(index_p !== -1){
```

Comment:
```suggestion
    while (index_p !== -1) {
```
while(공백)(조건)(공백){ ... } 식으로 작성해주세요. for문과 if문도 마찬가지 입니다. for(공백)(let i = 0; i < n; i++)(공백){
이런 식으로 작성 해주시는게 좋아요